src/ingestor/typedData/index.ts

```diff
@@ -40,18 +40,24 @@ export default async function ingestor(body) {
   let type = hashTypes[hash];
 
   let network = '1';
+  let aliased = false;
   if (!['settings', 'alias', 'profile'].includes(type)) {
     if (!message.space) return Promise.reject('unknown space');
     const space = await getSpace(message.space);
     if (!space) return Promise.reject('unknown space');
     network = space.network;
+    if (space.voting?.aliased) aliased = true;
   }
 
   // Check if signing address is an alias
+  const aliasTypes = ['follow', 'unfollow', 'subscribe', 'unsubscribe', 'profile'];
+  const aliasOptionTypes = ['vote', 'vote-array', 'vote-string', 'proposal', 'delete-proposal'];
```

Comment:
Hey @bonustrack, asking out of curiosity - why do you differentiate here between `aliasTypes` and `aliasOptionTypes`?

Reply:
`aliasTypes` define the actions that can do with alias, `aliasOptionTypes` define the actions that can use alias only if we enable the "aliased" space settings.

Reply:
At first I was confused too, maybe better to add a comment in code

Reply:
How about using `aliasedTypes` and `aliasableTypes` to differentiate between the ones that are always aliased and can be aliased optionally? :thinking:

Reply:
@zzuziak The actions "follow", "unfollow" etc can also be cast with the original address so technically both follow or vote actions can be called as aliasable, that's why I've added "option" in aliasOption but there must be better wording.

Reply:
Got it. I think it can be done in the var name without a comment. Something with `toggle`, `aliasToggableTypes`? Or simply `optional` instead of `option`